proto/google/events/cloud/build/v1/data.proto

```diff
@@ -0,0 +1,626 @@
+  //
+  // Default time is ten minutes.
+  google.protobuf.Duration timeout = 12;
+
+  // A list of images to be pushed upon the successful completion of all build
+  // steps.
+  //
+  // The images are pushed using the builder service account's credentials.
+  //
+  // The digests of the pushed images will be stored in the `Build` resource's
+  // results field.
+  //
+  // If any of the images fail to be pushed, the build status is marked
+  // `FAILURE`.
+  repeated string images = 13;
+
+  // TTL in queue for this build. If provided and the build is enqueued longer
+  // than this value, the build will expire and the build status will be
+  // `EXPIRED`.
+  //
+  // The TTL starts ticking from create_time.
+  google.protobuf.Duration queue_ttl = 40;
+
+  // Artifacts produced by the build that should be uploaded upon
+  // successful completion of all build steps.
+  Artifacts artifacts = 37;
+
+  // Google Cloud Storage bucket where logs should be written (see
+  // [Bucket Name
+  // Requirements](https://cloud.google.com/storage/docs/bucket-naming#requirements)).
+  // Logs file names will be of the format `${logs_bucket}/log-${build_id}.txt`.
+  string logs_bucket = 19;
+
+  // A permanent fixed identifier for source.
+  SourceProvenance source_provenance = 21;
+
+  // The ID of the `BuildTrigger` that triggered this build, if it
+  // was triggered automatically.
+  string build_trigger_id = 22;
+
+  // Special options for this build.
+  BuildOptions options = 23;
+
+  // URL to logs for this build in Google Cloud Console.
+  string log_url = 25;
+
+  // Substitutions data for `Build` resource.
+  map<string, string> substitutions = 29;
+
+  // Tags for annotation of a `Build`. These are not docker tags.
+  repeated string tags = 31;
+
+  // Secrets to decrypt using Cloud Key Management Service.
+  repeated Secret secrets = 32;
+
+  // Stores timing information for phases of the build. Valid keys
+  // are:
+  //
+  // * BUILD: time to execute all build steps
+  // * PUSH: time to push all specified images.
+  // * FETCHSOURCE: time to fetch source.
+  //
+  // If the build does not specify source or images,
+  // these keys will not be included.
+  map<string, TimeSpan> timing = 33;
+}
+
+message Source {
+  // Location of source.
+  oneof source {
+    // If provided, get the source from this location in Google Cloud Storage.
+    StorageSource storage_source = 2;
+
+    // If provided, get the source from this location in a Cloud Source
+    // Repository.
+    RepoSource repo_source = 3;
+  }
+}
+
+// Location of the source in an archive file in Google Cloud Storage.
+message StorageSource {
+  // Google Cloud Storage bucket containing the source (see
+  // [Bucket Name
+  // Requirements](https://cloud.google.com/storage/docs/bucket-naming#requirements)).
+  string bucket = 1;
+
+  // Google Cloud Storage object containing the source.
+  //
+  // This object must be a gzipped archive file (`.tar.gz`) containing source to
```

Comment:
Remove this sentence, given that the event is providing information rather than making requirements of callers?